### DISCUSSION TITLE
Using trivy to scan filesystem for (non-Node) javascript

### DISCUSSION THREAD

**Discussion by tschroeder13** [aquasecurity/trivy#7097] [Q&A]
Hey all,
would it be possible to use trivy to scan a filesystem for non-NodeJS javascript files similar as [retireJS](https://github.com/RetireJS/retire.js) does?
I'd like to use trivy for that, as I managed to implement a client/server mode and can distribute the client's binary via network filesystem - no need to install additional software to my servers.

Thanks in advance for your feedback
BR
Tobias

**Comment by DmitriyLewen** [comment]
Hello @tschroeder13 

Trivy doesn't scan source code.
We only scan dependency files (such as lock file).

You can see list of supported files [here](https://aquasecurity.github.io/trivy/v0.53/docs/coverage/language/).

Regards, Dmitriy
